src/form-field/internal.tsx

```diff
@@ -115,7 +117,11 @@ export default function InternalFormField({
 
       {(constraintText || errorText) && (
         <div className={styles.hints}>
-          {errorText && <FormFieldError id={slotIds.error}>{errorText}</FormFieldError>}
+          {errorText && (
+            <FormFieldError id={slotIds.error} errorIconAriaLabel={i18nStrings?.errorIconAriaLabel}>
```

Comment:
Just notice one more thing: the id is attached to the text span. Does it make sense to move the id up to cover icon and the text?